### PR TITLE
Add a clarification about sdk upload classifiers

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/24371

## Description
Upload classifiers using demisto-sdk is working on 6.0.0 version and up.
Added a clarification for it in the README file and in the help option in sdk upload.

## Must have
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -421,7 +421,8 @@ def format_yml(input=None, output=None, from_version=None, no_validate=None):
 @main.command(name="upload",
               short_help="Upload integration to Demisto instance. DEMISTO_BASE_URL environment variable should contain"
                          " the Demisto server base URL. DEMISTO_API_KEY environment variable should contain a valid "
-                         "Demisto API Key.")
+                         "Demisto API Key."
+                         " * Notes: Upload classifiers works on Cortex XSOAR from version 6.0.0 and up.*")
```
Comment (ShahafBenYakir) [inline demisto_sdk/__main__.py:425]:
```suggestion
                         " * Note: Uploading classifiers works on Cortex XSOAR from version 6.0.0 and up.*")
```

File: demisto_sdk/commands/upload/README.md [hunk 2]
```diff
@@ -23,6 +23,9 @@ Supported content entities:
 - Layouts
 - Classifiers
 
+#### Notes
+Upload classifiers works on Cortex XSOAR from version 6.0.0 and up.
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/upload/README.md:27]:
```suggestion
#### Limitation:
Uploading classifiers work on Cortex XSOAR from version 6.0.0 and up.
```

File: demisto_sdk/__main__.py (hunk 1 above)
Comment (barchen1) [reply demisto_sdk/__main__.py]: done

File: demisto_sdk/commands/upload/README.md (hunk 2 above)
Comment (barchen1) [reply demisto_sdk/commands/upload/README.md]: done

**Pull request merged**